app/src/main/java/com/mobiledevelopmentfoodapp/dto/Food.kt

```diff
@@ -1,10 +1,13 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Food(var name: String, var description: String, var price: Double, @SerializedName("id") var productId:Int = 0 ) {
+data class Food(
+    internal var name: String,
+    internal var description: String,
+    internal var price: Double,
+    @SerializedName("id")internal var productId: Int = 0,
```

Comment:
Added trailing comma. A trailing comma is a stylistic choice. The Kotlin style guide recommends a trailing comma. Best practices for trailing commas can be found here. https://kotlinlang.org/docs/coding-conventions.html#trailing-commas